cluster/internal/data.pb.go

```diff
@@ -17,15 +17,17 @@ It has these top-level messages:
 package internal
 
 import proto "github.com/gogo/protobuf/proto"
+import fmt "fmt"
 import math "math"
 
 // Reference imports to suppress errors if they are not otherwise used.
 var _ = proto.Marshal
+var _ = fmt.Errorf
 var _ = math.Inf
 
 type WriteShardRequest struct {
-	ShardID          *uint64  `protobuf:"varint,1,req" json:"ShardID,omitempty"`
-	Points           [][]byte `protobuf:"bytes,2,rep" json:"Points,omitempty"`
+	ShardID          *uint64  `protobuf:"varint,1,req,name=ShardID" json:"ShardID,omitempty"`
```

Comment:
I updated my protobuf dependencies.  They had a lot of code churn.  These are a result of running `go generate ./...`.  The build process will also make these changes when creating the binaries as I believe we get the latest packages when building.  It really is showing that we don't have an answer to vender versioning on our project.